org.khelekore:prtree:jar:1.5.0 pom.xml not found when trying to build a plugin importing PlotSquared-Core

### Server Implementation

Paper

### Server Version

1.18

### Describe the bug

when building:
```
[INFO] Scanning for projects...
[INFO] 
[INFO] -----------------< co.purevanilla.mcplugins:ShopWorld >-----------------
[INFO] Building ShopWorld 1.0.0
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for org.khelekore:prtree:jar:1.5.0 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.463 s
[INFO] Finished at: 2021-12-09T17:02:05+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project ShopWorld: Could not resolve dependencies for project co.purevanilla.mcplugins:ShopWorld:jar:1.0.0: Failure to find org.khelekore:prtree:jar:1.5.0 in https://papermc.io/repo/repository/maven-public/ was cached in the local repository, resolution will not be reattempted until the update interval of papermc has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

Process finished with exit code 1
```

not being able to find the dependency ``org.khelekore:prtree:jar:1.5.0``


### To Reproduce

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>co.purevanilla.mcplugins</groupId>
    <artifactId>ShopWorld</artifactId>
    <version>1.0.0</version>

    <properties>
        <maven.compiler.source>17</maven.compiler.source>
        <maven.compiler.target>17</maven.compiler.target>
    </properties>

    <repositories>
        <repository>
            <id>papermc</id>
            <url>https://papermc.io/repo/repository/maven-public/</url>
        </repository>
        <repository>
            <id>intellectualsites snapshots</id>
            <url>https://mvn.intellectualsites.com/content/repositories/snapshots/</url>
        </repository>
    </repositories>

    <dependencies>
        <dependency>
            <groupId>io.papermc.paper</groupId>
            <artifactId>paper-api</artifactId>
            <version>1.18-R0.1-SNAPSHOT</version>
            <scope>provided</scope>
        </dependency>
        <dependency>
            <groupId>com.plotsquared</groupId>
            <artifactId>PlotSquared-Core</artifactId>
            <version>6.1.4</version>
            <scope>provided</scope>
        </dependency>
    </dependencies>
</project>
```
then
```
maven clean install
```

### Expected behaviour

compile success

### Screenshots / Videos

_No response_

### Error log (if applicable)

_No response_

### Plot Debugpaste

api usage / plugin coding, no debugpaste

### PlotSquared Version

6.1.4

### Checklist

- [X] I have included a Plot debugpaste.
- [X] I am using the newest build from https://www.spigotmc.org/resources/77506/ and the issue still persists.

### Anything else?

_No response_

--- a/Bukkit/build.gradle.kts
+++ b/Bukkit/build.gradle.kts
@@ -94,7 +94,7 @@ tasks.named<ShadowJar>("shadowJar") {
 tasks {
     withType<Javadoc> {
         val opt = options as StandardJavadocDocletOptions
-        opt.links("https://papermc.io/javadocs/paper/1.18/")
+        opt.links("https://papermc.io/javadocs/paper/1.17/")
         opt.links("https://docs.enginehub.org/javadoc/com.sk89q.worldedit/worldedit-core/7.2.7/")
         opt.links("https://docs.enginehub.org/javadoc/com.sk89q.worldedit/worldedit-bukkit/7.2.7/")
         opt.links("https://jd.adventure.kyori.net/api/4.9.3/")

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -73,7 +73,7 @@ allprojects {
 
     plugins.withId("java") {
         the<JavaPluginExtension>().toolchain {
-            languageVersion.set(JavaLanguageVersion.of(17))
+            languageVersion.set(JavaLanguageVersion.of(16))
         }
     }
 

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -26,7 +26,7 @@ hyperverse = "0.6.0-SNAPSHOT"
 mvdwapi = "3.1.1"
 
 # Third party
-prtree = "1.5.0"
+prtree = "2.0.0"
 aopalliance = "1.0"
 pipeline = "1.4.0-SNAPSHOT"
 arkitektonika = "2.1.0"
@@ -75,7 +75,7 @@ essentialsx = { group = "net.essentialsx", name = "EssentialsX", version.ref = "
 hyperverse = { group = "se.hyperver.hyperverse", name = "Core", version.ref = "hyperverse" }
 
 # Third party
-prtree = { group = "org.khelekore", name = "prtree", version.ref = "prtree" }
+prtree = { group = "com.intellectualsites.prtree", name = "PRTree", version.ref = "prtree" }
 aopalliance = { group = "aopalliance", name = "aopalliance", version.ref = "aopalliance" }
 pipeline = { group = "com.intellectualsites", name = "Pipeline", version.ref = "pipeline" }
 mvdwapi = { group = "be.maximvdw", name = "MVdWPlaceholderAPI", version.ref = "mvdwapi" }

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -1,10 +1,10 @@
 [versions]
 # Minecraft expectations
-gson = "2.8.8" # Version set by Minecraft
+gson = "2.8.0" # Version set by Minecraft
 log4j-api = "2.14.1" # Version set by Minecraft
 
 # Platform expectations
-paper = "1.18-R0.1-SNAPSHOT"
+paper = "1.17.1-R0.1-SNAPSHOT"
 checker-qual = "3.19.0"
 guice = "5.0.1"
 findbugs = "3.0.1"

